tests/auto/tst_dbmanager/tst_dbmanager.pro

```diff
@@ -1,5 +1,6 @@
 TARGET = tst_dbmanager
```

Comment:
Is this accidentally here or really needed for tst_dbmanager.pro?

Needed at least for tst_declarativetabmodel.pro and tst_webview.pro I'd think. Clean build is failing.